src/main/scala/com/gu/dynamodbswitches/Switch.scala

```diff
@@ -1,9 +1,20 @@
 package com.gu.dynamodbswitches
 
+import com.amazonaws.services.dynamodbv2.model.AttributeValue
+
 case class Switch(name: String, default: Boolean = false) {
   @volatile private var state: Boolean = default
 
   def enabled: Boolean = state
 
   def enabled_=(value: Boolean) = state = value
+
+  def toStringAttribute() = {
+    new AttributeValue().withS(this.name)
+  }
+
+  def toNumAttributeValue() = {
```

Comment:
This seems like an odd method. It converts the default value to a string containing 1 or 0.

Perhaps a better name might be `defaultAsAttributeValue`? 

But even better might be to inline this logic, or define a more generic and pure method in a companion object. E.g.

    def asAttributeValue(b: Boolean) = {
      val str = if(b) "1" else "0"
      new AttributeValue().withN(str)
    }